tests/test_plugins/test_plugin/server.py

```diff
@@ -77,6 +78,13 @@ def rawWithDecorator(self, params):
         return b'this is a raw response'
 
     @access.public
+    @rawResponse
+    @describeRoute(None)
+    def rawReturningText(self, params):
+        setResponseHeader('Content-Type', 'text/plain')
```

Comment:
cherrypy defaults to `text/html;charset=utf-8`, at least in its current version.